### ISSUE TITLE
Pypi release version mismatch.

### ISSUE THREAD

**Issue by alferguet** [hjson/hjson-py#21]
Pypi release is 3.0.1 uploaded the Oct 27, 2017 but the latest github release is 3.6.5 on Oct 24, 2014.

Which release is the most updated and why both sites don't have the same version? Thanks in advance.

**Comment by Foxboron** [comment]
Yo,

Currently packaging up `qmk` for Arch, and it depends on this. Would it be possible to get an explanation why the version mismatch exists?

**Comment by dqsully** [comment]
GitHub is the latest release. I guess I didn't understand how Pypi does updates. @laktak could you add me as a maintainer on Pypi (username dqsully).

**Comment by laktak** [comment]
@dqsully  I've sent you an invite.

**Comment by dqsully** [comment]
@Foxboron It looks like this version mismatch is because this project started as a fork of simplejson, so it imported all the Git tags from that project. As of writing this v3.0.1 is the latest version, and I'm working on rolling out v3.0.2 right now. I'll clean up the Git tags here in a bit too.

**Comment by dqsully** [comment]
@laktak I got the invite, but it doesn't look like I'm able to upload a package update. Are there any controls on your side for that?

**Comment by Foxboron** [comment]
@dqsully Awesome. When you cleanup the tags and release `v3.0.2` I'd strongly recommend adding a CHANGELOG explaining the mismatch.

It doesn't seem like anyone has packaged the github releases https://repology.org/project/python:hjson/versions

**Comment by laktak** [comment]
@dqsully nope, though I see you created a token:
```
API token added
Permissions: Can upload to this project
Controlled by: dqsully
Token name: Mad**** PC 
```

**Comment by dqsully** [comment]
Ok finally figured it out. 3.0.2 is released!

**Comment by laktak** [comment]
@dqsully did you see this? https://travis-ci.org/github/hjson/hjson-py/builds/727355454 

I guess it's time to remove the outdated python versions 2.6 & 3.3

**Comment by dqsully** [comment]
sffc fixed that already for this most recent release. I think TravisCI just re-ran those old commits since I fixed all the git tags, but v3.0.2 worked perfectly

**Comment by Foxboron** [comment]
Thanks for fixing up the repository :) Makes it way less confusing
